bsp4j/src/main/java/ch/epfl/scala/bsp4j/Protocol.xtend

```diff
@@ -530,6 +531,39 @@ class RunResult {
   }
 }
 
+@JsonRpcData
```

Comment:
These should be in a separate JvmExtension file, since JVM is language/environment-specific, and wouldn't need to be supported by all build servers

Reply:
Done